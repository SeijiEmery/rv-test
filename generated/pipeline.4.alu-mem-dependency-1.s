sd s0, 0(s1)
	ld s2, 0(s1)
	add s4, s3, s2
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
