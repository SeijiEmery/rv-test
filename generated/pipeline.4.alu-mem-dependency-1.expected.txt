tests/pipeline.test.s:43
R8 = 420
R9 = 3072
R18 = 420
R19 = 69
R20 = 489
